kapeta.yml

```diff
@@ -5,6 +5,10 @@ metadata:
   description: Provide JWT authentication to your services
   visibility: public
 spec:
+  icons:
+    - type: url
+      value: https://storage.googleapis.com/kapeta-public-cdn/icons/themed/json-web-tokens-dark.svg
+      theme: light
```

Comment:
The icon name ends with `-dark.svg` but is used in `theme: light` 🤔 

Do we not want the colorful logo in light mode?

![image](https://github.com/kapetacom/provider-resource-internal-auth-jwt/assets/1045799/3f62391a-5173-4f89-8ea1-abf22423c0d0)